src/Message/ApplePayCompleteAuthorizeRequest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+/**
+ * After the Apple Pay payment sheet is fully loaded on the frontend. The user can authorize a payment 
+ * using Touch or Face ID –– this will grant access to the ApplePayPaymentToken. You can then authorize a payment
+ *  using this request with the ApplePayPaymentToken as a parameter along with other information needed for
+ *  an authorization (see AuthorizeRequest). No money will be transferred during authorization.
+ * After authorizing a transaction, call capture to complete the transaction and transfer the money.
+ *
+ * You may use other fields from the ApplePayPayment object to fill out billing info.
+ * This request only requires the 'token' field from the ApplePayPaymentToken extracted from the ApplePayPayment object.
```

Comment:
can you please specify the name of the parameter that this should be passed in as like the other requests do? also could you mention that that object comes from the frontend?

Reply:
It does, it's called 'token'. I can change 'field' to 'parameter' to make it more clear.

Reply:
i read this as "take the 'token' field from the ApplePayPayment object and pass it to this request", but that doesn't make it clear that the name of the parameter for this request is _also_ token. the other request objects have lists of parameters that the request takes, starting with the parameter name, which i think is more clear.

Reply:
I can add this line right after this one:
`Pass the extracted 'token' to the 'applePayToken' parameter of this class.`

Look okay to you?

Reply:
yup!